webapp/src/main/resources/locale/outlook/Outlook.xml

```diff
@@ -58,7 +58,7 @@
     <pathAddDescription>Create a new folder</pathAddDescription>
     <pathOpenDescription>Open in new window</pathOpenDescription>
     <savingAttachment>Saving your files...</savingAttachment>
-    <savedAttachment>Your files were successfully saved</savedAttachment>
+    <savedAttachment>Your files were successfully saved in</savedAttachment>
```

Comment:
We already have i18n string for `in` text - `<in>in</in>`, is it not used? This text should not be changed and if but not this way to fix it.

Reply:
<in>in</in> isn't used anywhere (about savedAttachment context). That's why the "in" added here.

Reply:
Fixed